wolfcrypt/src/ecc.c

```diff
@@ -12436,6 +12436,23 @@ int wc_ecc_ctx_set_peer_salt(ecEncCtx* ctx, const byte* salt)
     return 0;
 }
 
+int wc_ecc_ctx_set_kdf_salt(ecEncCtx* ctx, const byte* salt, word32 len)
```

Comment:
Can you add at least an inline function comment here? Also important to note the salt passed must be declared static or something allocated from heap and retained, since the pointer is used directly (not a copy of it).

Reply:
Fixed.